packages/documentation/src/styles/rafiki.css

```diff
@@ -54,12 +65,60 @@
   text-align: center;
 }
 
-.sl-markdown-content :is(h1, h2, h3, h4, h5, h6) > a {
-  color: inherit;
-  text-decoration: none;
-  transition: color 0.2s ease-in-out;
+/* Overflowing table styles */
+.overflow-table {
+  overflow-x: scroll;
+  background:
```

Comment:
This adds a bit of a curved shadow that only shows up when there is something to scroll, to indicate to users that there is something to scroll. Turns off when you scroll to the end.